src/components/matches/MatchesGrid.vue

```diff
@@ -164,6 +168,11 @@ export default defineComponent({
       type: Boolean,
       required: true,
     },
+    showHeroes: {
+      type: Boolean,
+      required: false,
+      default: true,
```

Comment:
How about we call it "show details" and it's off by default in order to keep the history clean, but allow users to expand if they want

Reply:
Isn't `showDetails` less specific than `showHeroes`, when it's only being used specifically for showing hero icons? There are no other details. If there are more details added later, we can change it then?

Reply:
Fine with me :)